test/minigzip.c

```diff
@@ -109,7 +109,7 @@ static char *strwinerror (error)
         LocalFree(msgbuf);
     }
     else {
-        sprintf(buf, "unknown win32 error (%ld)", error);
+        sprintf(buf, "unknown win32 error (%u)", error);
```

Comment:
%u or %lu?

Reply:
DWORD is unsigned int (yeah I know) on windows

Reply:
> DWORD is unsigned int

DWORD is unsigned long

Reply:
https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/262627d8-3418-4627-9218-4ffe110850b2

```c
 typedef unsigned long DWORD, *PDWORD, *LPDWORD;
```